lib/private/shell.ts

```diff
@@ -67,6 +83,27 @@ export async function shell(command: string[], options: ShellOptions = {}): Prom
   });
 }
 
+function handleShellOutput(
+  chunk: Buffer | string,
+  options: ShellOptions,
+  eventType: ShellEventType
+): void {
+  switch (options.subprocessOutputDestination) {
+    case 'ignore':
+      return;
+    case 'publish':
+      options.shellEventPublisher(eventType, chunk.toString('utf-8'));
+      break;
+    case 'stdio':
+    default:
+      if (eventType != 'data_stderr') {
```

Comment:
This now would print the command line as well, but the old code didn't use to do that. I think you only want to print for `data_*` events.

Reply:
It sent the command line to the logger. Did the logger always send it to `stdout`?

Reply:
I fixed this yeah the only things that went to stdout was the shell stdout. That is supposed to go to stderr, good catch

Reply:
Nice, thanks. Please reflect on why I needed to tell you this twice.